### ISSUE TITLE
Problem with nesting inside Field

### ISSUE THREAD

**Issue by andrewQwer** [davidkpiano/react-redux-form#111]
Hi, I am trying to use Field with validation and have some problems. My default layout looks like this:
```html
<Field model="editUser.firstName"
  validators={{required: function(val) { return val && val.length > 5}}}
  validateOn="blur">
  <div className="flex-row">
    <label htmlFor="firstName">First name</label>
    <input type="text" id='firstName'/>
  </div>                      
</Field>
```
When I am trying to put validation error inside `<div className="flex-row">` like this:
```html
                       <div className="flex-row">
                            ... <input field here>
                            { getField(self.props.editUserForm, 'firstName').errors.required &&
                            <div>First name is required</div>
                            }
                        </div>
``` 
I get the following error in console:
![image](https://cloud.githubusercontent.com/assets/710513/14203258/b0d0f814-f805-11e5-8ed5-409656432600.png)
But If I place validation error right inside Field everything works fine:
```html
<Field...>
   <div>
       <input fields>
   </div>
{ getField(self.props.editUserForm, 'firstName').errors.required &&
                        <div>First name is required</div>}
</Field>
```

**Comment by davidkpiano** [comment]
Seems like a React/JS issue and not a RRF issue. Can you post your full working code?

**Comment by andrewQwer** [comment]
Look at this gist: https://gist.github.com/andrewQwer/27bc9e647c01e926e9e6d0fe77d40815
When you change input text you will get the following error: `field-component.js:336 Uncaught TypeError: Cannot read property 'props' of null`
If you remove `<div id="error-wrapper">` and place `getField` right after  input element it starts working.

**Comment by davidkpiano** [comment]
Ah yes, that is a bug. I just need to add a null check. Will fix soon!

**Comment by davidkpiano** [comment]
Fixed in `0.9.6`!
